src/preprocess/query_generator.py

```diff
@@ -0,0 +1,225 @@
+import json
+
+
+def remove_all_unnecessary_dependencies(f):
+    f.write("delete from programstageinstance_messageconversation;" + "\n")
+    f.write("delete from programinstancecomments;" + "\n")
+    f.write("delete from programinstanceaudit;" + "\n")
+    f.write("delete from datavalueaudit;" + "\n")
+    f.write("delete from trackedentitydatavalueaudit;" + "\n")
+    f.write("delete from trackedentityattributevalueaudit;" + "\n")
+    f.write("delete from programstageinstance_messageconversation;" + "\n")
+    f.write("delete from dataapprovalaudit;" + "\n")
+    f.write("delete from interpretation_comments;" + "\n")
+    f.write("delete from interpretationcomment;" + "\n")
+    f.write("delete from messageconversation_messages;" + "\n")
+    f.write("delete from messageconversation_usermessages;" + "\n")
+    f.write("delete from messageconversation;" + "\n")
```

Comment:
To ease readability and copy/paste, I'd use `"""` to build a multiline string and a final `f.write`. Note that we can still use string interpolation with `"""  .... """.format(...)`